src/Sulu/Bundle/TestBundle/Behat/DefaultContext.php

```diff
@@ -0,0 +1,153 @@
+<?php
+/*
+ * This file is part of the Sulu CMS.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\TestBundle\Behat;
+
+use Behat\Behat\Context\SnippetAcceptingContext;
+use Behat\Behat\Hook\Scope\BeforeScenarioScope;
+
+/**
+ * Default context class for Sulu contexts.
+ */
+class DefaultContext extends BaseContext implements SnippetAcceptingContext
```

Comment:
Yeah. http://docs.behat.org/en/latest/guides/2.definitions.html#definition-snippets

Means that you can autogenerate the code (or snippet) for the step definitions